tests/testthat/test-print.R

```diff
@@ -2,25 +2,25 @@
 context("print")
 
 test_that("short", {
-  skip_if_offline()
-
-  x <- ps("csardi")
+  vcr::use_cassette("ps_csardi", {
+    x <- ps("csardi")
+  })
   out <- capture.output(print(x))
   expect_equal(length(out), 12)
   expect_match(out[1], "csardi.*packages in.*seconds")
   expect_match(out[2], "#\\s+package\\s+version\\s+by\\s+@\\s+title")
-  expect_match(out[3], "100\\s+igraph.*Gábor Csárdi.*Network Analysis")
+  expect_match(out[3], "100\\s+igraph.*Gábor Csárdi.*Network Analysis", useBytes = TRUE)
```

Comment:
this is the thing I needed to make the test pass on Appveyor, not sure why. :upside_down_face: